src/en/actions.md

```diff
@@ -13,141 +12,138 @@ against the schema defined in actions.yaml. See
-    properties: {}
-    title: report
-    type: object
-run:
-  description: Run some command
+add-repo: Create a git repository.
+add-repo-user: Give a user permissions to access a repository.
+add-user: Create a new user.
+get-repo: Return the repository's path.
+list-repo-users: List all users who have access to a repository.
+list-repos: List existing git repositories.
+list-user-repos: List all the repositories a user has access to.
+list-users: List all users.
+remove-repo: Remove a git repository.
+remove-repo-user: Revoke a user's permissions to access a repository.
+remove-user: Remove a user.
+```
+
+To show the full schema for all the actions on a service, append the `--schema`
+argument to the `list-actions` command. For example, here's the beginning of
+the output from `juju list-actions git --schema`:
+
+```bash
+add-repo:
+  additionalProperties: false
+  description: Create a git repository.
   properties:
-    command:
-      default: pwd
-      description: The command to run
+    repo:
+      description: Name of the git repository.
       type: string
-    title: run
-    type: object
+  required:
+  - repo
+  title: add-repo
+  type: object
+...
 ```
 
 > Note that the full schema is under the `properties` key of the root Action.
 > Juju Actions rely on [JSON-Schema](http://json-schema.org) for validation.
 > The top-level keys shown for the Action (`description` and `properties`) may
 > include future additions to the feature.
 
-### `juju action do`
-
-Trigger an action (the ID of the action will be displayed, for use with `juju
-action fetch <ID>` or `juju action status <ID>`; see below):
+### `juju run-action`
 
-```bash
-juju action do mysql/0 backup
-Action queued with id: <ID>
+Trigger an action. This command takes the unit as an argument and returns an ID
+for the action. The ID can used with `juju show-action-output <ID>` or `juju
+show-action-status <ID>.
 
-juju action do mysql/0 tpcc
-ERROR action "tpcc" not defined on unit "mysql/0"
+If an action requires parameters, these can be passed directly. For example, we
+could create a new 'git' repository by triggering the 'add-repo' action and
+following this with the name we'd like to give the new repository:
 
-juju action do mysql/0 benchmark
-Action queued with id: <ID>
 
-juju action do mysql/0 restart
-Action queued with id: <ID>
+```bash
+juju run-action git/0 add-repo repo=myproject
 ```
-
-Parameters can be passed directly:
+This will return the ID for the new action:
 
 ```bash
-juju action do mysql/0 benchmark bench="foo"
-Action queued with id: <ID>
-
-juju action do mysql/0 benchmark bench.kind="foo" bench.speed="fast"
-Action queued with id: <ID>
+Action queued with id: 3a7cc626-4c4c-4f00-820f-f881b79586d10
 ```
 
-Or indirectly via a YAML file (those provided directly take precedence):
+You can also set parameters indirectly via a YAML file, although you can
+override the parameters within the file by providing them directly.
 
  > Example params.yml
```

Comment:
the '>' for markdown has no effective style in docs, so we don't use it (except when other people write stuff, apparently). I would suggest making this an h4, but they look rubbish, so maybe just italicise or something?